scripts/LPSRE/increasing-observability-operator-pvc-size/metadata.yaml

```diff
@@ -0,0 +1,41 @@
+file: script.sh
+name: increasing-observability-operator-pvc-size # name should be the same as the subdir, eg: SRE/example in this case
+description: Increase the size of the PVCs used by the Observability Operator running in rhacs-observability namespace
+author: bseref
+allowedGroups: 
+  - LPSRE
+rbac:
+    roles: []
+    clusterRoleRules:
```

Comment:
Should be able to use role/rolebinding here instead of cluster level, as seems all the operations are happened in `rhacs-observability` namespace

Reply:
nice suggestion! I've limited the permissions to namespace scope only